pom.xml

```diff
@@ -104,6 +104,14 @@
             <scope>system</scope>
             <systemPath>${project.basedir}/lib/LibsDisguises-9.0.9.jar</systemPath>
         </dependency>
+
```

Comment:
There must be a repo with Core Protect in... Please try and find an official repo for Core Protect and use that, adding more local files to the POM just digs a deeper hole for us to try and climb out of.